### DISCUSSION TITLE
Timeline on the editor has a big zoom-in by default

### DISCUSSION THREAD

**Discussion by gustarmartins** [ppy/osu#16573] [Ideas] [answered]
On stable, one of the only reasons mappers zoom in the timeline is when they use rare beat snap divisor as 1/8, 1/12, 1/16 so they can actually see the extra snaps.

As of right now, the default zoom of the Lazer feels REALLY fast, and you can zoom-out, but it won't apply globally, which means you have to do it for everytime you open the editor. That can get quite annoying.

I suggest that the timeline zoom matches the stable behaviour by default, and maybe adding an option to set a default zoom according to each mapper preference would be great.

Current default on Lazer -> https://i.imgur.com/3VIkqjQ.png
Current default on Stable -> https://user-images.githubusercontent.com/45127556/150661025-2fc07d69-0dba-4e4e-b1b0-15653b8848a8.png

What it should be the default on Lazer, imo -> https://i.imgur.com/INTrtL5.png

That way the details of each object (bpm, slider velocity, hs sample) still do not overlap each other, and the zoom looks way better to what every mapper is used to.
It will overlap on stuff snapped to the 1/4, though, so that might be an issue worth looking into.

**Comment by peppy** [comment, marked as answer]
Can definitely agree with adjusting the default zoom, but also adding save support for it. Tracking via https://github.com/ppy/osu/issues/16581.
